### DISCUSSION TITLE
Unable to run sample code

### DISCUSSION THREAD

**Discussion by jee-mj** [dlemstra/Magick.NET#1244] [Help]
I am trying to build the sample code to see it in action as-per [this sample folder](https://github.com/dlemstra/Magick.NET/tree/main/samples/Magick.NET.Samples) since the code in the [docs folder](https://github.com/dlemstra/Magick.NET/tree/main/docs) doesn't seem to work.
![image](https://user-images.githubusercontent.com/28581723/188317568-f97b8a86-e038-4854-87d6-43539d196d1e.png)

My console app is setup like that, but for some reason after I exit from the cli, I don't see any output files.

**Comment by dlemstra** [comment]
That is because none of the sample code is actually called in that project? You will need to call the code yourself.
